src/content/web3/index.md

```diff
@@ -0,0 +1,148 @@
+
+### What is web3? {#what-is-web3}
+
+Web3 has become a catch-all term for the vision of a new, better internet. At its core, Web3 uses blockchain, cryptocurrency, and NFTs to give power back to the users in the form of ownership. Chris Dixon said it best: web1 was read-only, web2 is read/write, web3 will be read/write/own.
+
+#### Core ideas of web3 {#core-ideas}
+
+Although it's challenging to provide a rigid definition what web3 is, a few core principles guide its creation.
+
+- **Web3 is decentralized:** instead of large swathes of the internet controlled and owned by centralized entities, ownership gets distributed amongst its builders and users.
+- **Web3 is permissionless:** everyone has equal access to participate in web3, and no one gets excluded.
+- **Web3 has native payments:** it uses cryptocurrency for spending and sending money online instead of relying on the outdated infrastructure of banks and payment processors.
+- **Web3 is trustless:** it operates using incentives and economic mechanisms instead of relying on trusted third-parties
+
+### Why is web 3.0 important? {#why-is-web3-important}
+
+So why does web3 matter? Web3's killer features aren't isolated and don't fit into neat categories. But, for simplicity, we've tried to separate them to make them easier to understand.
+
+#### Ownership {#ownership}
+
+Web3 gives you ownership of your digital assets in an unprecedented way. For example, say you're playing a web2 game. If you purchase an in-game item, it is tied directly to your account. If the game creators delete your account, you will lose these items. Or, if you stop playing the game, you lose the value you invested into your in-game items.
+
+Web3 allows for direct ownership through [non-fungible tokens (NFTs)](/nft/). No one, not even the game's creators, has the power to take away your ownership. And, if you stop playing, you can sell or trade your in-game items on open markets and recoup their value.
+
+<ButtonLink to="/nft/">
+  More on NFTs
+</ButtonLink>
+
+#### Governance {#governance}
+
+As a Web2 platform increases in popularity, the power of the individual user's opinion decreases. Simultaneously, the power of the platform's governing body also increases. Over time, this power imbalance can spiral out of control until the platform can ignore grievances with no repercussions.
+
+Web3 empowers users to participate in the decision making of a platform through tokens. A recent example of this is Ethereum Name Service (ENS), which distributed 25% of its voting rights to existing users by sending ENS tokens to their wallets. Anyone with ENS tokens can vote on proposals about the future of ENS. ENS is a decentralized autonomous organization (DAO) because its ownership gets distributed amongst its token holders.
+
+_In Web2, you can refuse to use a platform; in Web3, you can vote to change it._
+
+<ButtonLink to="/dao/">
+  More on DAOs
+</ButtonLink>
+
```

Comment:
I'm not a huge fan of this section.  Governance as described is perfectly possible within the context of Web2, and something that doesn't function like a DAO may still be a completely viable web3 application.  I think governance is orthogonal to the ideals of web3.  The same patterns can (and probably will) emerge within the context of web3.

Many things will be built by someone who chooses to ignore their users.  Web3 is about giving user's the power to "vote with their feet" without having to leave behind the content they provided (because they own their own data, and data is permissionless), not about constraining the types of mechanisms that builders use to decide what they want to build.

Reply:
Agree with this critique!

The final sentence feels a bit problematic to me - I think one of the big problems with web2 is that, because the small group of web2 mega-platforms are so deeply embedded in mainstream web infrastructure, trying to boycott them is ostracizing for most users. Also in web3 the power of individual votes to influence platforms seems overrated. 

The value of web3 (imho) is that you can walk away from a specific web3 platform with your data at any time with minimal consequences for your overall web3 experience. Also permissionlessness and open source by default discourages apps becoming too monopolistic because alternative versions can be built quickly from forks (e.g. uni/sushi).

Reply:
Would overall agree with the comments here. How a particular platform governs itself is kinda separate to what makes something web3... I would honestly just consider dropping this section